source/agora/test/Base.d

```diff
@@ -232,7 +232,8 @@ public const(PublicKey)[] makeTestNetwork (
             registry.register(conf, "normal");
         }
 
-        return key_pairs.map!(a => a.address).array;
+        return new Network(NodeConfig.init, node_configs.map!(
+                               c => format("http://%s:%d", c.address, c.port)).array);
```

Comment:
Is this some kind of auto-indentation thing emacs does? It's over the 80-char limit, and to me this looks better (and simpler to code, it's just a tab hit):

```
return new Network(NodeConfig.init, node_configs.map!(
    c => format("http://%s:%d", c.address, c.port)).array);
```

Reply:
Yep I have to configure this. Fixed.